<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupMainWindow</class>
 <widget class="QMainWindow" name="GroupMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>其他</string>
    </property>
    <addaction name="actionabout"/>
    <addaction name="actionaboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>用户</string>
    </property>
    <addaction name="actionuserInfo"/>
    <addaction name="actionloginOut"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockWidget_left">
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid gray</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeView" name="treeView">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px;
border-top:1px solid gray  </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionabout">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/mainWin/Icon/about_1102px_1143785_easyicon.net.png</normaloff>:/mainWin/Icon/about_1102px_1143785_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionaboutQt">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/mainWin/Icon/assistant_qt_256px_1171940_easyicon.net.png</normaloff>:/mainWin/Icon/assistant_qt_256px_1171940_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="actionuserInfo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/mainWin/Icon/group.png</normaloff>:/mainWin/Icon/group.png</iconset>
   </property>
   <property name="text">
    <string>用户信息</string>
   </property>
  </action>
  <action name="actionloginOut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/mainWin/Icon/Logout_225px_1198327_easyicon.net.png</normaloff>:/mainWin/Icon/Logout_225px_1198327_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>登出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
